src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -915,7 +915,7 @@ private String getNameFromRoster(String participantIdentifier, boolean isFullNam
     /**
      * Get the displayable full name from an email.
      * 
-     * This function is different from getNameForEmail as it obtains the name
+     * <p>This function is different from {@link #getNameForEmail} as it obtains the name
      * using the class roster, instead of from the responses.
      * @param participantIdentifier
```

Comment:
The `JavadocParagraph` rule claims that it can enforce such, but obviously it's buggy. Perhaps future versions of Checkstyle (7.x) has fixed it.
(if you're wondering why we're stuck at version 6.19, it's because of Java 8)